aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Forms.xml

```diff
@@ -1,6 +1,10 @@
 <Namespace Name="Microsoft.AspNetCore.Components.Forms">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides classes for managing form views, state, and validations.</summary>
+    <remarks>
+      For more information, see 
+      <see href="https://docs.microsoft.com/aspnet/core/blazor/forms-validation">
+      ASP.NET Core Blazor forms and validation</see>.
```

Comment:
```suggestion
      For more information, see  <see href="https://docs.microsoft.com/aspnet/core/blazor/forms-validation">  ASP.NET Core Blazor forms and validation</see>.
```

Reply:
I don't think there's supposed to be newlines here. @JamesNK  will correct me if I'm wrong.

Reply:
It won't matter either way. Might as well update for consistency with other xml files.